packages/history-service/src/internal/history-multiplexer.ts

```diff
@@ -108,4 +117,17 @@ export class HistoryMultiplexer {
       state: newConsumerStates
     });
   }
+
+  private setConsumerStates(
```

Comment:
This method is more of a helper function and does not require access to the `this` object. So wouldn't a function be better suited?

Reply:
done